src/app/beer_garden/api/http/schemas/v1/login.py

```diff
@@ -0,0 +1,15 @@
+from marshmallow import Schema, fields
+
+
+class LoginInputSchema(Schema):
```

Comment:
I don't use these schemas for anything interesting other than the swagger docs.  I'd rather move them into brewtils at the point that we actually need them there, just for ease of development.